libs/interfaces/UploadOptions.ts

```diff
@@ -1,5 +1,27 @@
 import { ReadStream } from "fs";
 
+interface TransformationObject {
+  type: "transformation";
+  value: string;
+}
+interface GifToVideoOrThumbnailObject {
+  type: "gif-to-video" | "thumbnail";
+  value?: string;
+}
+
+interface AbsObject {
+  type: "abs";
+  value: string;
+  protocol: "hls" | "dash";
+}
+
+type postTransformation = TransformationObject | GifToVideoOrThumbnailObject | AbsObject;
+
+interface transformation{
+  pre?: string
+  post?: postTransformation[]
+}
+
```

Comment:
Interfaces should be named with proper CamelCasing

Reply:
done